test/ix-chart/2207.1.0/questions.yaml

```diff
@@ -23,9 +23,64 @@ groups:
     description: "Specify resources to be allocated to workload"
   - name: "Resource Limits"
     description: "Set CPU/memory limits for Kubernetes Pod"
-
+  - name: "Portal Configuration"
+    description: "Configure UI web portal"
 
 questions:
+  - variable: enableUIPortal
+    label: "Enable WebUI Portal (only supported in TrueNAS SCALE Bluefin)"
+    description: "Enable webui portal for easier access to workload ( Only valid for TrueNAS SCALE Bluefin )"
+    group: "Portal Configuration"
+    schema:
+      type: boolean
+      default: false
+  - variable: portalDetails
+    label: "WebUI Portal"
+    description: "Configure WebUI Portal"
+    group: "Portal Configuration"
+    schema:
```

Comment:
might be best to put a dict under this, for easy multi-portal support in the future.
This extra dict layer doesn't have to be used yet, but gives a lot more flexibility.